sirepo/smtp.py

```diff
@@ -94,14 +95,14 @@ def _init():
         _cfg.server = "not " + _DEV_SMTP_SERVER
         _SEND = _send_directly
         return
-    _SEND = _send_with_auth
+    _SEND = _send_to_configured_smtp_server
```

Comment:
`_send_via_relay_server`. Config probably should be relay_server, but leave as is.

Reply:
8f9c96785